src/main/java/com/example/demo/constants/Const.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.constants;
+
+public interface Const {
```

Comment:
Rename class. Use full words representation. Split file. Do not store all constants in one file.